actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/BulkUploadBackGroundJobActor.java

```diff
@@ -114,10 +114,12 @@ private void process(Request actorMessage) {
       }
       if (((String) dataMap.get(JsonKey.OBJECT_TYPE)).equalsIgnoreCase(JsonKey.USER)) {
         long startTime = System.currentTimeMillis();
-        ProjectLogger.log("processUserInfo start at : " + startTime, LoggerEnum.INFO.name());
+        ProjectLogger.log(
+            "BulkUploadBackGroundJobActor:processUserInfo start at : " + startTime,
+            LoggerEnum.INFO.name());
         processUserInfo(jsonList, processId, (String) dataMap.get(JsonKey.UPLOADED_BY));
         ProjectLogger.log(
-            "total time taken to processUserInfo for processId  : "
+            "total time taken by BulkUploadBackGroundJobActor:processUserInfo for processId  : "
```

Comment:
change  total time taken by BulkUploadBackGroundJobActor:processUserInfo for processId  : to  "BulkUploadBackGroundJobActor:processUserInfo  Total time taken : "